onnxruntime/test/optimizer/nhwc_transformer_test.cc

```diff
@@ -455,6 +455,40 @@ TEST(NhwcTransformerTests, ConvSplit) {
   }
 }
 
+TEST(NhwcTransformerTests, ConvPad) {
+  std::vector<std::string> pad_modes = {"constant", "reflect", "edge"};
+  for (const auto& mode : pad_modes) {
+    auto build_test_case = [&](NhwcTestHelper& helper) {
+      auto* input_arg = helper.MakeInput<uint8_t>({1, 23, 13, 13});
+      auto* conv1_output_arg = helper.MakeIntermediate();
+      auto* pads_const = helper.MakeScalarInitializer<uint8_t>(131);
+      auto* pads_arg = helper.MakeInitializer<int64_t>({8LL}, {0, 0, 1, 1, 0, 0, 1, 1});
```

Comment:
use helper.Make1DInitializer?

Reply:
Would it be better to make the pads be unique per dimension to make sure they are copied over in the right order (like 0, 0, 1, 2, 0, 0, 3, 4)?

Reply:
fixed

Reply:
Yeah, it is better.